gateway-developer-plugin/src/main/java/com/ca/apim/gateway/cagatewayconfig/CAGatewayDeveloper.java

```diff
@@ -110,7 +110,7 @@ private static BuildEnvironmentBundleTask createBuildEnvironmentBundleTask(@NotN
     private static BuildFullBundleTask createBuildFullBundleTask(@NotNull Project project, GatewayDeveloperPluginConfig pluginConfig, BuildDeploymentBundleTask buildDeploymentBundleTask) {
         // Create build-full-bundle task
         final BuildFullBundleTask buildFullBundleTask = project.getTasks().create(BUILD_FULL_BUNDLE, BuildFullBundleTask.class, t -> {
-            t.getEnvironmentConfig().set(pluginConfig.getEnvironmentConfig().getOrElse(Collections.emptyMap()));
+            t.getEnvironmentConfig().set(pluginConfig.getEnvironmentConfig());
```

Comment:
@uppoju ,we are not making use of this property to create env entities bundle, I merged the changes to feature branch as part of env bundle story.

Reply:
This is for defect fix DE457376

Reply:
@RaghuReddy if we are not supporting backward compatibility then its not required to fix this

Reply:
@uppoju , we are going to support that, but that should inline with exiting change (configFolder and configName)

Reply:
@uppoju , I have addressed this in PR : https://github.com/CAAPIM/gateway-developer-plugin/pull/232
Now environment entities can be picked from configFolder as well as environmentConfig, have a look at the PR